app/views/pull_requests/show.html.erb

```diff
@@ -107,4 +112,18 @@
 
 
 <%= link_to 'Edit', edit_repository_pull_request_path(id: @pull_request.id) %> |
-<%= link_to 'Back', repository_pull_requests_path %> | 
\ No newline at end of file
+<%= link_to 'Back', repository_pull_requests_path %> | 
+
+<div class="status-container">
+  <% if @pull_request.status == 'Open' && @have_conflicts == false %>
+    <%= link_to 'Merge pull request', merge_repository_pull_request_path(user_name: find_user.name, repository_id: current_repository.title, id: @pull_request), class: 'btn btn-success', method: 'POST' %>
+    <%= link_to "Close pull request", close_repository_pull_request_path(user_name: find_user.name, repository_id: current_repository.title, id: @pull_request), class: 'btn btn-danger', method: 'POST'%>
+  <% elsif @pull_request.status == 'Open' && @have_conflicts %>
+    <div class="btn btn-danger">Have conflict, Please push your compare branch again before rebase your compare branch to base branch and resolve conflicts.</div>

```

Comment:
Have conflict, Please push your compare branch again before rebase your compare branch to base branch and resolve conflicts. 
->
There is conflict between two branch. Please resolve the conflict locally and try again.